plugins/flytekit-deck-standard/flytekitplugins/deck/renderer.py

```diff
@@ -53,12 +53,21 @@ class FrameProfilingRenderer:
     Generate a ProfileReport based on a pandas DataFrame
     """
 
-    def __init__(self, title: str = "Pandas Profiling Report"):
+    def __init__(self, title: Optional[str] = None):
```

Comment:
API wise, there are now two places to set kwargs for `ydata_profiling.ProfileReport`, in `__init__` (for only title) and in `to_html`.

By looking at the `Renderable` protocol, it looks like the design was to have configuration passed in `__init__`, because the `to_html` to restricted to one input:

https://github.com/flyteorg/flytekit/blob/master/flytekit/deck/renderer.py#L17-L23

Reply:
Yeah, I saw that, but I don't think we quite adhering to the protocol everywhere, for example the [pandera renderer](https://github.com/flyteorg/flytekit/blob/a6d6335819b05a4f7082d8bdea8827763e21515c/plugins/flytekit-pandera/flytekitplugins/pandera/pandas_renderer.py#L254-L259).

This is an indication that all invocations of `to_html` happen in user-code, so the protocol is "just" a convention? Can we make it more strict (and if so, I think we should add kwargs to the protocol).